searchstats.py

```diff
@@ -115,7 +115,8 @@ def main():
   for line in search_log:    # Iterate through lines of the log file.
     if first_line:    # If this is the first line, we need to figure out what
                       # kind of file we've received (search log or partnerlog).
-      if len(((line.split())[0]).split('.')) == 4:
+      if (len(((line.split())[0]).split('.')) == 7)or(len(((line.split())[0]).split('.')) == 4):
```

Comment:
please add a spaces on either side of the "or" and split into 2 lines (for readability)